docs/rules/space-unary-ops.md

```diff
@@ -72,6 +74,9 @@ new[foo][0];
 
 delete(foo.bar);
 
+// ReferenceError: "deletefoo" is not defined (space is required here for correct syntax)
+deletefoo.bar;
+
```

Comment:
Fair enough. I removed the `deletefoo` vs `delete foo` examples since the rule doesn't apply to them. There isn't a section for `"words": false` and I don't know that it's worth adding just for this. I think the note and better-aligned examples is probably enough to address the confusion.

Reply:
Agreed, thanks!